__tests__/node_fs_test.ml

```diff
@@ -0,0 +1,10 @@
+open Jest
+
+let _ =
+  describe "content" (fun _ ->
+      test "non-zero length" (fun _ ->
+          Just (Truthy (Js.Undefined.bind [%node __filename] begin fun [@bs] f ->
```

Comment:
`f` is a name typically used for functions. `file`, `filename` or `name` would be better names.